tools/compile_tf_graph.py

```diff
@@ -268,7 +381,7 @@ def init_op(self):
       :rtype: tf.Operation
       """
       assert self.var_initial_value is not None
-      return tf.assign(self.var, self.var_initial_value, name="init_state_var_%s" % self.name).op
+      return tf.assign(self.var, self.var_initial_value, name="init_state_var_%s" % self.name, validate_shape=False).op
```

Comment:
Why is that `validate_shape` needed? It should not be needed I think.

Reply:
forgot why, but it's needed during some test before with the attention model config

Reply:
Remove it again. It should not be needed. Or document why it really is needed.